crates/turborepo-lib/src/turbo_json/mod.rs

```diff
@@ -101,6 +101,7 @@ impl From<&RawRemoteCacheOptions> for ConfigurationOptions {
             timeout: remote_cache_opts.timeout,
             upload_timeout: remote_cache_opts.upload_timeout,
             enabled: remote_cache_opts.enabled,
+            no_update_notifier: None, // Remote cache options don't include this
```

Comment:
Can you remove this line? This is already done by the `..Self::default()` on the following line.